hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -44,13 +44,20 @@ func (pkgDef *PackageDefinition) EmitDefinitions(outputDir string) {
 	allocateTypesToFiles(otherDefinitions, filesToGenerate)
 	emitFiles(filesToGenerate, outputDir)
 	emitGroupVersionFile(pkgDef, outputDir)
+
+	return len(filesToGenerate)
```

Comment:
You may get a conflict here with my PR where I added some error handling/reporting here.

Reply:
Yup. Merged.